Submitted By:            Bruce Dubbs <bdubbs_at_linuxfromscratch_dot_org>
Date:                    2011-11-19
Initial Package Version: 0.2.2
Upstream Status:         Not submitted
Origin:                  Upstream and LFS
Description:             This patch contains removes nis references that 
                         have been removed from glibc.

diff -Naur libtirpc-0.2.2.orig/src/Makefile.am libtirpc-0.2.2/src/Makefile.am
--- libtirpc-0.2.2.orig/src/Makefile.am 2011-05-02 07:10:40.000000000 -0500
+++ libtirpc-0.2.2/src/Makefile.am      2011-11-19 19:43:59.000000000 -0600
@@ -50,7 +50,7 @@
         rpc_callmsg.c rpc_generic.c rpc_soc.c rpcb_clnt.c rpcb_prot.c \
         rpcb_st_xdr.c svc.c svc_auth.c svc_dg.c svc_auth_unix.c svc_generic.c \
         svc_raw.c svc_run.c svc_simple.c svc_vc.c getpeereid.c \
-        auth_time.c auth_des.c authdes_prot.c des_crypt.c
+        auth_time.c

 ## XDR
 libtirpc_la_SOURCES += xdr.c xdr_rec.c xdr_array.c xdr_float.c xdr_mem.c xdr_reference.c xdr_stdio.c
diff -Naur libtirpc-0.2.2.orig/src/rpc_soc.c libtirpc-0.2.2/src/rpc_soc.c
--- libtirpc-0.2.2.orig/src/rpc_soc.c   2011-05-02 07:10:40.000000000 -0500
+++ libtirpc-0.2.2/src/rpc_soc.c        2011-11-19 19:46:22.000000000 -0600
@@ -519,6 +519,7 @@
  * Create the client des authentication object. Obsoleted by
  * authdes_seccreate().
  */
+#ifdef NEVER
 AUTH *
 authdes_create(servername, window, syncaddr, ckey)
        char *servername;               /* network name of server */
@@ -546,6 +547,7 @@
        dummy = authdes_seccreate(servername, window, NULL, ckey);
        return (dummy);
 }
+#endif

 /*
  * Create a client handle for a unix connection. Obsoleted by clnt_vc_create()

